c-deps/rocksdb-0001-Fix-NDEBUG-windows.patch

```diff
@@ -0,0 +1,90 @@
+diff --git a/port/win/env_win.cc b/port/win/env_win.cc
+index 8099f84..1e4a917 100644
+--- a/port/win/env_win.cc
++++ b/port/win/env_win.cc
+@@ -1077,13 +1077,19 @@ std::string Env::GenerateUniqueId() {
+   UuidCreateSequential(&uuid);
+ 
+   RPC_CSTR rpc_str;
+-  auto status = UuidToStringA(&uuid, &rpc_str);
+-  assert(status == RPC_S_OK);
```

Comment:
The duplicate calls are a mild maintainability concern. Easy to change one without changing the other. And sprinkling `#ifdef NDEBUG` is a bit unsightly.